npm/zip.js

```diff
@@ -0,0 +1,33 @@
+var shell = require('shelljs'),
+  path = require('path'),
+  async = require('async'),
+  fs = require('fs'),
+  codegen_path,
+  commandOut,
+  pwd = shell.pwd();
+
+const codegen = process.argv[2],
+PATH_TO_CODEGENS_FOLDER = path.resolve(__dirname, '../codegens');
+
+// throw JS error when any shell.js command encounters an error
+shell.config.fatal=true;
+
+codegen_path = path.join(PATH_TO_CODEGENS_FOLDER, codegen);
+
+try {
+    exists=fs.statSync(codegen_path).isDirectory();
+} catch (err) {
+    console.log(`Codegen ${codegen} doesn't exist, please enter a valid name`);
+    shell.exit(1);
+}
+if (exists) {
+    console.log(`${codegen} : zip`);
+    shell.pushd(codegen_path, "q");
+    commandOut = shell.exec('npm pack --color always');
```

Comment:
What is the need of `--color always`?

Reply:
@umeshp7 Because by default shellJS doesn't display the result of exec() in colors as done by Chalk. 
This has been highlighted here https://github.com/shelljs/shelljs/issues/86